lib/cartopy/io/__init__.py

```diff
@@ -383,6 +385,43 @@ def fetch_raster(self, projection, extent, target_resolution):
         raise NotImplementedError()
 
 
+class LocatedImageRasterSource(RasterSource):
+    """Expose a simple image as a Raster Source object."""
+    def __init__(self, image, projection):
+        """
+        Create a Raster Source of a geo-located simple image.
+
+        :param image: `LocatedImage` instance
+        An image, expressed as a LocatedImage object (containing an image and
+        an extent.
```

Comment:
Missing parenthesis (!)
```suggestion
            an extent).
```